rules/windows/persistence_group_modification_by_system.toml

```diff
@@ -0,0 +1,67 @@
+[metadata]
+creation_date = "2024/06/26"
+integration = ["system", "windows"]
+maturity = "production"
+updated_date = "2024/06/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a user being added to an active directory group by the SYSTEM (S-1-5-18) user. This behavior can indicate
+that the attacker has achieved SYSTEM privileges in a domain controller, which attackers can obtain by exploiting
+vulnerabilities or abusing default group privileges (e.g., Server Operators), and is moving to a domain account.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.security*", "logs-windows.forwarded*"]
+language = "eql"
+license = "Elastic License v2"
+name = "AD Group Modification by SYSTEM"
+risk_score = 47
+rule_id = "6f024bde-7085-489b-8250-5957efdf1caf"
+severity = "medium"
+tags = [
+    "Domain: Endpoint",
+    "OS: Windows",
+    "Use Case: Threat Detection",
+    "Tactic: Persistence",
+    "Use Case: Active Directory Monitoring",
+    "Data Source: Active Directory",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+iam where winlog.api == "wineventlog" and event.action == "added-member-to-group" and
+winlog.event_data.SubjectUserSid : "S-1-5-18" and
```

Comment:
This should exclude known AD managed service accounts (MSA) as well or do we want to include those?

Reply:
I expect the MSAs to have a different SID, as this one should be limited to Local System